packages/client/src/forms/register/mappings/query/registration-mappings.ts

```diff
@@ -97,28 +97,40 @@ export const certificateDateTransformer =
 
 export const convertToLocal = (
   mobileWithCountryCode: string,
-  alpha3CountryCode: string
+  alpha3CountryCode?: string
 ) => {
   /*
    *  If country is the fictional demo country (Farajaland), use Zambian number format
    */
 
-  const countryCode = countryAlpha3toAlpha2(alpha3CountryCode)
+  if (!mobileWithCountryCode) {
+    return
+  }
```

Comment:
Please move this condition to the [call location](https://github.com/opencrvs/opencrvs-core/blob/afe6cd327419cee634f131098271da41f1f7bd62/packages/client/src/forms/register/mappings/query/registration-mappings.ts#L135) of this function instead. This is so that validation of parameters would always be done as early in the callstack as possible. Even better would be if `mobileWithCountryCode`'s type wouldn't allow empty strings, but probably not easy to achieve here